app/views/fines/index.html.erb

```diff
@@ -0,0 +1,29 @@
+<h1>Bills</h1>
+<p>NOTE: Fines for items that are still checked out will not appear here until the items have been returned</p>
+<table class="table">
+  <caption>Bills Table</caption>
+  <thead>
+  <tr>
+    <th scope="col">Title</th>
+    <th scope="col">Author</th>
+    <th scope="col">Reason</th>
+    <th scope="col">You owe</th>
+    <th scope="col">Date billed</th>
+  </tr>
+  </thead>
+
+  <tbody>
+    <% @fines.each do |fine| %>
+    <tr>
+      <th scope="row"><%= fine.title %></th>
```

Comment:
I assume title will include link to record, might need to confirm with Ruth maybe 🤔

Reply:
Oh good catch, no, let's just fix it. I'll add it in.

Reply:
Looks like item type is under title too. My mistake.

Reply:
fixed in latest